app/diagnostics.go

```diff
@@ -731,6 +734,16 @@ func (a *App) trackPlugins() {
 		"disabled_backend_plugins":      backendDisabledCount,
 		"plugins_with_settings":         settingsCount,
 		"plugins_with_broken_manifests": brokenManifestCount,
+		"enable_antivirus":              pluginData(pluginTrackData, "antivirus"),
```

Comment:
As discussed offline the list needs to get updated.

Reply:
I've added `webex` to the list of plugins to track `enabled` and `version`. Let me know if anymore that should be added @aaronrothschild @hanzei